src/en/components/header/design.md

```diff
@@ -24,13 +24,13 @@ date: "git Last Modified"
 
 **The search element** supports including a search that is local or global.
 
-<img class="b-sm b-gray mb-500 p-400" src="/images/en/anatomy/gcds-header-anatomy.svg" alt=""/>
+<img class="b-sm b-gray mb-500 p-400" src="/images/en/anatomy/gcds-header-anatomy.svg" alt="Home link Taxonomy has five parts. The skip to content - indicated by a bright blue double white lined box with label of Skip to main content. The Phase banner - indicated by a navy blue lined box with label of stages of the site. The stage 'pilot' is in a white navy blue outlined box. GC header banner with a Canadian Flag with Government of Canada  and Government du Canada. Language toggle pointing at Français to use to toggle to French. Top bar pointing at Home link in a light grey banner with Site menu link outlined by a dark grey link to indicate the selected link with unselected Site menu link with no line underneath it"/>
```

Comment:
Just flagging: is this using the correct slot names?

Reply:
This will need to be updated to match newer anatomy images